### ISSUE TITLE
ZEROTIER - OPENWRT

### ISSUE THREAD

**Issue by SuoaJ** [mwarning/zerotier-openwrt#43]
I installed OpenWRT 18.06.1 on a Linksys EA3500 router. According to the OpenWRT page on the device the package is "arm_xscale". Then I proceed to install Zerotier but I get this error below:

`
root@OpenWrt:/tmp/juke# opkg install zerotier
Installing zerotier (1.2.4-2) to root...
Downloading http://downloads.openwrt.org/snapshots/trunk/kirkwood/generic/packages/packages/zerotier_1.2.4-2_kirkwood.ipk.
Configuring zerotier.
Generate secret - please wait...
Segmentation fault
cat: can't open '/tmp/zt.sample_config.secret': No such file or directory
rm: can't remove '/tmp/zt.sample_config.secret': No such file or directory
`

Can anyone help? Thanks.

**Comment by SuoaJ** [comment]
I should also note that before this error in the post above, I tried installing Zerotier and it complained about a kernel dependency as shown below. I did a "opkg --force-depends" which eventually installed Zerotier but ended up with the error above.

`
root@OpenWrt:~# opkg install zerotier
Installing zerotier (1.2.4-2) to root...
Downloading http://downloads.openwrt.org/snapshots/trunk/kirkwood/generic/packag                                                                                                                     es/packages/zerotier_1.2.4-2_kirkwood.ipk.
Collected errors:
 * satisfy_dependencies_for: Cannot satisfy the following dependencies for zerot                                                                                                                     ier:
 *      kernel (= 4.4.14-1-82b00ff3637317d0c0f26a8376fb1881) *
 * opkg_install_cmd: Cannot install package zerotier.

`
